source/physics/elasticity/kinematics.inst.in

```diff
@@ -0,0 +1,68 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2016 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+
+for (deal_II_dimension : DIMENSIONS; number : REAL_SCALARS)
+{
+    template
+    Tensor<2,deal_II_dimension,number>
+    Physics::Elasticity::Kinematics::F<deal_II_dimension,number>(const Tensor<2,deal_II_dimension,number>&);
```

Comment:
I think I dimly remember that in C++98 you can only instantiate stuff in its proper namespace, not by referencing functions in a nested namespace. So you have to enter and leave the namespaces. But I may also be wrong. Let's just see what happens when this gets committed.